### PR TITLE
nginx-1.27.3-RELEASE

### PR DESCRIPTION
**Pull request by pluknet**

### CODE REVIEW

File: docs/xml/nginx/changes.xml [hunk 1]
```diff
@@ -5,6 +5,71 @@
...
+might be handled incorrectly on DragonFlyBSD.
+</para>
+</change>
+
+<change type="bugfix">
+<para lang="ru">
+в обработке ошибок в директиве proxy_store.
+</para>
+<para lang="en">
+in error handling in the "proxy_store" directive.
```
Comment (arut) [inline docs/xml/nginx/changes.xml]: It does not feel exactly like error handling, but I don't have a better version

File: docs/xml/nginx/changes.xml [hunk 2]
```diff
@@ -5,6 +5,71 @@
...
+the "resolve" parameter.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+директивы "resolver" и "resolver_timeout" в блоке upstream.
+</para>
+<para lang="en">
+the "resolver" and "resolver_timeout" directives in the "upstream" block.
```
Comment (arut) [inline docs/xml/nginx/changes.xml]: No quotes in the Russian version.

File: docs/xml/nginx/changes.xml [hunk 3]
```diff
@@ -5,6 +5,71 @@
...
+</para>
+</change>
+
+<change type="change">
+<para lang="ru">
+теперь протоколы TLSv1 и TLSv1.1 по умолчанию запрещёны.
```
Comment (arut) [inline docs/xml/nginx/changes.xml]: запрещЕны

File: docs/xml/nginx/changes.xml (hunk 1 above)
Comment (pluknet) [reply docs/xml/nginx/changes.xml]: I used wording from changes for 248baf426, but perhaps you are right, I will replace it with our usual "bugfixes in ..".

File: docs/xml/nginx/changes.xml (hunk 2 above)
Comment (pluknet) [reply docs/xml/nginx/changes.xml]: Fixed, thanks.

**Comment by pluknet** [comment]
- updated proxy_store description
- removed quotes from translation
- DragonFly BSD spelling
- fixed typo

**Comment by pluknet** [comment]
Added changes in the IMAP and realip/proxy_bind functionality.

**Review by y82 — APPROVED** [review]

**Comment by pluknet** [comment]
Merged realip/proxy_bind items.

**Review by arut — APPROVED** [review]
LGTM

**Pull request merged**